gaps/gap-1_gap_process/gap-1_gap_process.md

```diff
@@ -0,0 +1,44 @@
+---
+gap: 1
+title: GAP Process
+description: Specification of Golem Amendment Proposal process.
+author: stranger80 (@stranger80)>
+status: Draft
+type: Meta
+---
+
+## Abstract
+We are introducing a Golem Amendment Proposal process to establish governance over changes proposed, considered and implemented in Golem ecosystem.
+
+## Motivation
+The GAP process responds to a number of needs:
+- Establish governance over features, specification/standards amendments and process improvements managed by Golem Factory.
+- Ensure visibility of features in the Golem pipeline - this is especially valuable for interested parties outside of Golem Factory.
+- Provide ability for Golem community to submit proposals for features in a controlled, systematic manner.
+
+## Specification
+
+### GAP content & format
+
+- A GAP shall be raised as a pull request in `golem-architecture` repo titled along the lines of `GAP-draft Title`, including all the relevant artifacts (eg. GAP summary). 
+- A GAP summary `gap-draft_title.md` shall be provided (following a [template](../gap-template.md)) in `./gaps/gap-draft_title/`.
+- The GAP pull request shall be included in [GAP project](https://github.com/golemfactory/golem-architecture/projects/1).
+
+### GAP workflow
+
+![gap workflow](gap_process.drawio.png "GAP Process")
+
+## Rationale
+The GAP process template has been designed on the basis of Ethereum's [EIP template](https://github.com/ethereum/EIPs/blob/master/eip-template.md) - as we feel that template ensures the right level of proposal specification details relevant for Golem ecosystem.
+
+## Backwards Compatibility
+As currently there is no (semi)formal process for governance of Golem feature and change proposals - no backward incompatibilities are introduced. 
```

Comment:
For the sake of completeness, there were GIPS proposed
https://github.com/golemfactory/golem-architecture/tree/master/GIPS
and also RFCs long time ago
https://github.com/golemfactory/golem-model/blob/master/market/doc/RFC-template.md
But they are not used. 
For GAPS to be used we need to come up with a workflow (process of sheparding them).
Like the one described here https://eips.ethereum.org/EIPS/eip-1#eip-work-flow
Probably we would like to bind this workflow to the TechCouncil meeting and TechCouncil discord channel for vetting ideas (asking if an idea is original to avoid wasting time on something that will be rejected based on prior research ideas).